src/main/resources/CurrencyExtra.properties

```diff
@@ -0,0 +1,192 @@
+# Copyright 2008 Google Inc.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# This file contains additional data about currencies which are not included in
+# the CLDR data.  Currently, this file contains four fields separated by "|".
+# Empty trailing fields can be omitted along with their separators.
+#
+# The first field is a "portable" currency symbol, which is intended to be
+# recognized in most locales.
+#
+# The second field contains zero or more flags separated by spaces.
+#
+# At most one of the following:
+#   SymPrefix     The currency symbol goes before the number, regardless of the
+#                 normal position for this locale.
+#   SymSuffix     The currency symbol goes after the number, regardless of the
+#                 normal position for this locale.
+#
+# At most one of the following:
+#   ForceSpace    Always add a space between the currency symbol and the number.
+#   ForceNoSpace  Never add a space after the currency symbol and the number
+#
+# The third field overrides the CLDR-derived currency symbol if present.
+#
+# The fourth field is the simple currency symbol, if present.
+#
+# The following were set to 10 originally, which is POSITION_FLAG without
+# POS_FIXED_FLAG.  As I understand the code, this would have no effect.
+#  CZK DKK EUR ILS ISK NOK RUB SEK
```

Comment:
maybe one more comment about how this is inherited from google? that said, do we really need it at all? we know google doesnt use org.gwtproject at this time (nor do they keep com.google.gwt up to date...)

Reply:
it is used for generating flags for currencyList which is used in numberFormat, i didnt want to change this and break NumberFormat at this point in time.